src/Servers/HttpSys/src/RequestProcessing/RequestContextOfT.cs

```diff
@@ -25,6 +25,8 @@ protected override async Task ExecuteAsync()
 
             try
             {
+                InitializeFeatures();
```

Comment:
An exception from InitializeFeatures used to trigger a 400 response, now it Aborts the connection without a response. That's probably OK because we don't know of any current exceptions from here, that check was added after we'd fixed some prior exceptions.